examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -314,6 +365,226 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
+          fillColor: '#f7f7f7',
+          weight: 1.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0.5
+        },
+        countyStyle: {
+          fillColor: '#f7f7f7',
+          weight: 1.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0.5
+        },
+        countyUpperStyle: {
+          fillColor: '#f7f7f7',
+          weight: 1.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0.5
+        },
+        cityStyle: {
+          fillColor: '#f7f7f7',
+          weight: 1.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0.5
+        },
+        hoverStyle: {
+          weight: 5,
+          color: '#666',
+          fillOpacity: 0.5
+        },
+        colors: [ '#ffffff', '#92d1e1', '#4393c3', '#2166ac', '#f4a582', '#d6604d', '#b2182b'],
+        sentimentColors: ['#ff0000', '#C0C0C0', '#00ff00']
+      };
+    }
+
+    function setInfoControlPointMap() {
+      //change the style of the polygons
+      $scope.styles = {
+        initStyle: {
+          weight: 0.5,
+          fillOpacity: 0,
+          color: 'white'
+        },
+        stateStyle: {
+          fillColor: '#f7f7f7',
+          weight: 0.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0
+        },
+        stateUpperStyle: {
+          fillColor: '#f7f7f7',
+          weight: 0.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0
+        },
+        countyStyle: {
+          fillColor: '#f7f7f7',
+          weight: 0.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0
+        },
+        countyUpperStyle: {
+          fillColor: '#f7f7f7',
+          weight: 0.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0
+        },
+        cityStyle: {
+          fillColor: '#f7f7f7',
+          weight: 0.5,
+          opacity: 1,
+          color: '#92d1e1',
+          fillOpacity: 0
+        },
+        hoverStyle: {
+          weight: 0.7,
+          color: '#666',
+          fillOpacity: 0
+        },
+        colors: [ '#ffffff', '#92d1e1', '#4393c3', '#2166ac', '#f4a582', '#d6604d', '#b2182b'],
+        sentimentColors: ['#ff0000', '#C0C0C0', '#00ff00']
+      };
+
+      function zoomToFeature(leafletEvent) {
+          if (leafletEvent)
+              $scope.map.fitBounds(leafletEvent.target.getBounds());
+      }
+
+      function onEachFeature(feature, layer) {
+        layer.on({
+          click: zoomToFeature
+        });
+      }
+
+      loadGeoJsonFiles(onEachFeature);
+
+      if ($scope.zoomfunction) {
+        $scope.zoomfunction()
+      }
+
+      $scope.zoomfunction = $scope.$on("leafletDirectiveMap.zoomend", function () {
+        if ($scope.map) {
+          $scope.status.zoomLevel = $scope.map.getZoom();
+          $scope.bounds = $scope.map.getBounds();
+          if ($scope.status.zoomLevel > 9) {
```

Comment:
could this showing polygon logic be shared by both countmap and pointmap?

Reply:
This will be solved by later code refracturing work.